inc/saimpls.h

```diff
@@ -127,6 +127,19 @@ typedef enum _sai_inseg_entry_attr_t
      */
     SAI_INSEG_ENTRY_ATTR_NEXT_HOP_ID,
 
+    /**
```

Comment:
Can we add this as the last attribute in the list (between pop_qos_mode and end).

Reply:
Moved to end of list. Thanks for the comment